.vscode/cspell.json

```diff
@@ -39,22 +46,35 @@
     "eventhub",
     "eventhubs",
     "hmac",
+    "incrementalcopy",
     "iothub",
     "keyvault",
+    "legalhold",
+    "maxresults",
+    "maxresults",
+    "maxsize",
     "msrc",
     "newtonsoft",
     "oidc",
     "pageable",
     "pkce",
     "pkcs",
     "posix",
+    "RAGRS",
     "reqwest",
+    "restype",
     "rustup",
     "seekable",
     "servicebus",
     "stylesheet",
     "typespec",
+    "undelete",
+    "undelete",
+    "Undelete",
+    "Undelete",
```

Comment:
Please remove duplicates. cSpell is also case-insensitive so no need to store both variations.

Reply:
And, to clarify, I think "undelete" is general purpose enough to stay here. Several of these are. Use your best judgement.